docs/examples/batch-to-online.ipynb

```diff
@@ -316,16 +316,16 @@
     "scaler = preprocessing.StandardScaler()\n",
     "\n",
     "for xi, yi in stream.iter_sklearn_dataset(datasets.load_breast_cancer()):\n",
-    "    xi = scaler.learn_one(xi)"
+    "    xi = scaler.fit_one(xi)"
```

Comment:
Not an error, we replaced the `fit_` convention by `learn_` (cf. #329). Other notebooks need to be fixed.

Reply:
> Not an error, we replaced the `fit_` convention by `learn_` (cf. #329). Other notebooks need to be fixed.

Sorry about that. I have fixed it now.

Reply:
No worries @mihir-thakkar-ai, thx for your help :)